tests/solidosdk.test.ts

```diff
@@ -0,0 +1,73 @@
+import { SolidoSDK, TX_STAGE } from '@/index';
+import { Connection, PublicKey } from '@solana/web3.js';
+import { ERROR_CODE, ERROR_CODE_DESC, ERROR_MESSAGE } from '@/constants';
+
+describe('SolidoSDK', () => {
+  const walletWithStSolTokenAccount = new PublicKey('2Vn1xSUTo292A3knejUeifjt2A3aGNqyn9Svy8Kx8i4J');
+  const cluster = 'testnet';
```

Comment:
I think we can put the cluster in constants. And depending on the selected cluster, substitute the necessary values - for the wallet address, rpc address, etc.